packages/test/test-end-to-end-tests/src/test/audience.spec.ts

```diff
@@ -229,4 +229,56 @@ describeFullCompat("Audience correctness", (getTestObjectProvider, apis) => {
 			"client2's audience should be removed",
 		);
 	});
+
+	it.skip("second client should see first client in audience when it connects immediately after", async () => {
```

Comment:
What is our process for ensuring we come back and enable tests like these? Do we have a work item filed in ADO?

Reply:
could create and ado, but i imagine we will immediately backport this fix, release, and then consume in 6.0, and main, so tracking would be just for accounting purposes, all the work should happen serially

Reply:
Yeah, that was my idea as well. If it somehow gets delayed, I'll make sure to create tracking ADO items.